app/views/plans/_benchmark_activities.html.erb

```diff
@@ -1,14 +1,35 @@
-<div class="row bg-light-gray p-1">
-  <div class="col">
-    <b>Benchmark <%= benchmark_id %>:</b> <%=
-    benchmarks.indicator_text(benchmark_id) %>
+<div data-controller="benchmark" data-target="benchmark.self" class="benchmark-container" data-action="mouseleave->benchmark#reset">
+  <div class="row bg-light-gray p-1 header">
+    <div class="col-11">
```

Comment:
I feel like it would be better for the benchmark#reset action on mouseleave to be on this component instead of on the entire benchmark component. I'm finding it a little disconcerting that I can move the mouse away (thus causing the confirm message to disappear) and then move it back and still see "Really Delete?".

Reply:
ahh good catch, will update 👍